tests/tensorflow/test_tensorflow2_autolog.py

```diff
@@ -146,6 +150,15 @@ def clear_fluent_autologging_import_hooks():
     mlflow.utils.import_hooks._post_import_hooks.pop("keras", None)
 
 
+@pytest.fixture(autouse=True)
+def clear_autologging_config():
```

Comment:
Clear the autologging config between tests so that, e.g. calling `mlflow.tensorflow.autolog(log_model_signatures=True)` in test case n doesn't leave signature logging enabled in test case n + 1, which could cause issues if n + 1 verifies that signature logging is disabled for a particular case (e.g. `test_keras_autolog_does_not_log_model_signature_when_mlflow_autolog_called`)